packages/components/src/Card/Card.css

```diff
@@ -56,6 +56,12 @@
 }
 
 .header {
+  display: flex;
+  flex-direction: row;
```

Comment:
Do we need to specify a flex-direction? Web flex-direction's [initial value should be row](https://jobber.atlassian.net/browse/JOB-61896).